src/parser/binary-operation-parser.ts

```diff
@@ -94,13 +112,50 @@ export function parseReferentialBinaryOperation(
   )
 }
 
+export function parseFilterObject(
+  obj: Readonly<FilterObject<any, any>>,
+  combinator: 'and' | 'or'
+): OperationNode {
+  return parseFilterList(
+    Object.entries(obj)
+      .filter(([, v]) => !isUndefined(v))
+      .map(([k, v]) =>
+        parseValueBinaryOperation(k, needsIsOperator(v) ? 'is' : '=', v)
+      ),
```

Comment:
nitpicky

```suggestion
    Object.entries(obj)
      .reduce((parsed, [k, v]) => {
        if (isUndefined(v)) return acc
        
        const op = needsIsOperator(v) ? 'is' : '='
        
        parsed.push(parseValueBinaryOperation(k, op, v))
        return parsed
      }, []),
```

Reply:
Hmm... I think the current one is cleaner. Isn't the `parsed` array also `any[]` here?

Reply:
Yeah, TypeScript doesn't play nice with reduce, you usually have to `as T` the initial value.
This was just a micro-optimization (single pass vs. two passes).